app/components/each-property/component.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+export function getPropertyInputType(property) {
```

Comment:
I think keeping this in separate function (where we can tweak much easier) is definitely the right call.  Initially tying the property type to the component type makes sense, but I could imagine that in the future we might want to add "hints" to the underlying schema.

Definitely :+1: on this for now which keeps things simple and makes tweaking in the future easy.